app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="container-fluid">
+
+  <div class="row">
+    <div class='col-md-4'>
+      <h3 class='header'>Jobs By Interest</h3>
+      <table class="tableSection interestLevel">
+        <thead>
+          <tr><th>Interest Level</th><th># of Jobs</th></tr>
+        </thead>
+        <tbody>
+          <% @jobs.each do |key, value| %>
+          <tr class=key_<%= key %>>
+            <td><%= key %></td>
+            <td><%= value %></td>
+          </tr>
+          <% end %>
+        </tbody>
+      </table></br>
+    </div>
+
+    <div class='col-md-4'>
+      <h3 class="header">Jobs by Location</h3>
+      <table class="tableSection jobsLocation">
+        <thead>
+          <tr><th>Location</th><th># of Jobs</th></tr>
+        </thead>
+        <tbody>
+          <% @locations.each do |key, value| %>
+          <tr class=key_<%= key %>>
+            <td><%= link_to key, companies_path(:location => key) %></td>
+            <td><%= value %></td>
+          </tr>
+          <% end %>
+        </tbody>
+      </table></br>
+    </div>
+
+    <div class='col-md-4'>
+      <h3 class="header">My Top Companies</h3>
+      <table class="tableSection averageInterest">
+        <tr><th>Company</th><th>Average Interest</th></tr>
+        <% @companies.each do |company| %>
+        <tr class=company_<%= company.first %>>
+          <td><%= link_to company.first, company_path(Company.find_by(name: company.first)) %></td>
```

Comment:
this is an example of having too much logic in your views. you're directly accessing your model. how can you move this logic out? does it belong in the controller? 

isn't this the same as... `company_path(company.first)`?